python/tests/test_regs.json

```diff
@@ -37,21 +37,54 @@
                     "-P 50000 --preserve_performance_counters --save_resume "
                 ]
             },
+            "(",
```

Comment:
Can you please try different loss functions as well?
https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Loss-functions